opensubtitles/common.go

```diff
@@ -0,0 +1,153 @@
+package opensubtitles
+
+import "time"
+
+//Subtitles
+type Subtitles struct {
+	TotalPages int    `json:"total_pages"`
+	TotalCount int    `json:"total_count"`
+	Page       int    `json:"page"`
+	Data       []Data `json:"data"`
+}
+//Uploader
```

Comment:
Codacy found an issue: [comment on exported type Uploader should be of the form "Uploader ..." (with optional leading article)](https://app.codacy.com/manual/TheForgotten69/go-opensubtitles/pullRequest?prid=6093402)